workbench-core/accounts/src/services/accountService.ts

```diff
@@ -81,17 +82,17 @@ export default class AccountService {
    *
    * @returns Account entry in DDB
    */
-  public async create(accountMetadata: { [key: string]: string }): Promise<{ [key: string]: string }> {
+  public async create(
+    accountMetadata: CreateAccountRequest & { environmentInstanceFiles: string }
```

Comment:
Should we add `environmentInstanceFiles` to the request?

Reply:
Good question. I didn't include `environmentInstanceFiles` because in `hostingAccountService.create`, the parameters does not include `environmentInstanceFiles`. This is only needed by `accountService.create`. So far our pattern has been to have the `request` object match with the service that is being used in the router. This continues that pattern. 

The flow is shown below

`setUpAccountRoutes` -> `hostingAccountService` -> `hostingAccountLifecycleService` -> `accountService`